### PR TITLE
Update the resiliency page to 2.0.0

### PR DESCRIPTION
**Pull request by bleskes**

### CODE REVIEW

**Comment by jasontedor** [comment]
LGTM.

File: docs/resiliency/index.asciidoc [hunk 1]
```diff
@@ -104,50 +104,59 @@ Further issues remain with the retry mechanism:
...
-elected as master will update the cluster state to indicate the result of the election. Other nodes will submit
-a join request to the newly elected master node. Instead of immediately processing the election result, the elected master
-node should wait for the incoming joins from other nodes, thus validating that the result of the election is properly applied. As soon as enough
-nodes have sent their joins request (based on the `minimum_master_nodes` settings) the cluster state is updated.
-{GIT}12161[#12161]
+The family of circuit breakers has greatly reduced the occurrence of OOM
+exceptions, but it is still possible to cause a node to run out of heap
+space.  The following issues have been identified:
 
+* Set a hard limit on `from`/`size` parameters {GIT}9311[#9311]. (STATUS: ONGOING)
```
Comment (clintongormley) [inline docs/resiliency/index.asciidoc]: Fixed in v2.1.0

File: docs/resiliency/index.asciidoc [hunk 2]
```diff
@@ -104,50 +104,59 @@ Further issues remain with the retry mechanism:
...
-space.  The following issues have been identified:
+This status page is a start, but we can do a better job of explicitly documenting the processes at work in Elasticsearch, and what happens in the case of each type of failure. The plan is to have a test case that validates each behavior under simulated conditions. Every test will document the expected results, the associated test code and an explicit PASS or FAIL status for each simulated case.
 
-* Set a hard limit on `from`/`size` parameters {GIT}9311[#9311]. (STATUS: ONGOING)
-* Prevent combinatorial explosion in aggregations from causing OOM {GIT}8081[#8081]. (STATUS: ONGOING)
-* Add the byte size of each hit to the request circuit breaker {GIT}9310[#9310]. (STATUS: ONGOING)
+[float]
+=== Do not allow stale shards to automatically be promoted to primary (STATUS: ONGOING)
+
+In some scenarios, after loss of all valid copies, a stale replica shard can be assigned as a primary. This can lead to
```
Comment (clintongormley) [inline docs/resiliency/index.asciidoc]: after the loss of all valid shard copies

File: docs/resiliency/index.asciidoc [hunk 3]
```diff
@@ -158,61 +167,61 @@ be added to a shard's mapping until confirmed by the master.
...
+used policy. The amount of memory used by the cache depends on two primary
+components - the values it stores and the keys associated with them. Calculating
+the memory footprint of the values is easy enough but the keys accounting is
+trickier to achieve as they are, by default, raw Lucene objects. This is largely
+not a problem as the keys are dominated by the values. However, recent
+optimizations in Lucene have changed the balance causing the filter cache to
+grow beyond it's size.
 
-While we are working on a longer term solution ({GIT}9176[#9176]), we introduced a minimum weight of 1k for each cache entry. This puts an effective limit on the number of entries in the cache. See {GIT}8304[#8304] (STATUS: DONE, fixed in v1.4.0)
+While we are working on a longer term solution ({GIT}9176[#9176]), we introduced
```
Comment (clintongormley) [inline docs/resiliency/index.asciidoc]: The longer term solution is in place

File: docs/resiliency/index.asciidoc [hunk 4]
```diff
@@ -158,61 +167,61 @@ be added to a shard's mapping until confirmed by the master.
...
-[float]
-=== Do not allow stale shards to automatically be promoted to primary (STATUS: ONGOING)
-
-In some scenarios, after loss of all valid copies, a stale replica shard can be assigned as a primary. This can lead to
-a loss of acknowledged writes if the valid copies are not lost but are rather temporarily isolated. Work is underway
-({GIT}14671[#14671]) to prevent the automatic promotion of a stale primary and only allow such promotion to occur when
-a system operator manually intervenes.
-
-== Completed
+Note: this has been solved by the move to Lucene's query cache. See {GIT}10897[#10897]
```
Comment (clintongormley) [inline docs/resiliency/index.asciidoc]: Ah, I see the note here

**Comment by clintongormley** [comment]
A couple of minor comments. Thanks for doing this!

**Comment by bleskes** [comment]
@clintongormley I pushed another update.

**Comment by clintongormley** [comment]
LGTM

**Pull request merged**